test/test_utils.py

```diff
@@ -225,3 +225,35 @@ def test_invalid_png_data(self):
                           imageinfo.identify, data)
         self.assertRaises(imageinfo.UnrecognizedFormat,
                           imageinfo.identify, data)
+
+
+class TestEncodeFilename(unittest.TestCase):
+
+    @unittest.skipIf(os.path.supports_unicode_filenames , "unicode is supported")
+    def test_EncodeFilenames_Nosupport(self):
+        self.assertEqual(b'\xe5\xb0\x8f\xe5\xae\xa4', util.encode_filename("小室"))
+        self.assertEqual(b'\xe5\xb0\x8f', util.encode_filename('小'))
+
+    @unittest.skipUnless(os.path.supports_unicode_filenames and sys.platform == 'darwin', "requires unicode support and darwin os")
```

Comment:
I don't understand why this test is specific to darwin systems.

Reply:
we need to cover that possibility for this line: https://github.com/metabrainz/picard/blob/99ef1b28a24eca762088e0f87150a0ed804261be/picard/util/__init__.py#L91